src/pages/costModelsDetails/costModelInfo/index.tsx

```diff
@@ -56,6 +56,7 @@ class CostModelInformation extends React.Component<Props, State> {
           <TabContent
             eventKey={0}
             id="refPriceList"
+            title="PriceList"
```

Comment:
Not sure how `TabContent` uses titles, but should be "Price list" for a title / aria attributes?

Reply:
Actually, I don't see `title` as a valid property for `TabContent`. Should be able to just remove it?